### PR TITLE
feat: add support for Python projects using pipenv

### PR DESCRIPTION
**Pull request by taleinat**
- [ ] Ready for review
- [ ] Follows CONTRIBUTING rules
- [ ] Reviewed by Snyk internal team

#### What does this PR do?
Add support for Python projects using pipenv.

#### Where should the reviewer start?
This is just many small changes in many files...

#### How should this be manually tested?
Run the tests in `test/acceptance/cli.acceptance.test.js`.

#### Any background context you want to provide?
Support for pipenv was [added](https://github.com/snyk/snyk-python-plugin/commit/52e2b57b) in [version 1.7.0 of the Python plugin](https://github.com/snyk/snyk-python-plugin/releases/tag/v1.7.0).

#### What are the relevant tickets?
[SC-4814](https://snyksec.atlassian.net/browse/SC-4814)

### CODE REVIEW

**Comment by adrukh** [comment]
@taleinat thanks! Please sign the CLA, we'll 'swallow' this PR into our internal release process (not a straight-forward merge).

**Comment by taleinat** [comment]
@adrukh, I've signed the CLA and fixed the JSCS failure.

File: lib/detect.js [hunk 1]
```diff
@@ -35,6 +36,7 @@ var DETECTABLE_PACKAGE_MANAGERS = {
   'build.sbt': 'sbt',
   'yarn.lock': 'yarn',
   'requirements.txt': 'pip',
+  'Pipfile': 'pipenv', // jscs:ignore disallowQuotedKeysInObjects
```
Comment (adrukh) [inline lib/detect.js]: I'd actually remove the quotes from the `Pipfile` key, but that's a matter of taste :)

Comment (taleinat) [reply lib/detect.js]: Done.

**Comment by adrukh** [comment]
@taleinat I'm closing this PR after having taken it 'behind the stage' to follow our release process. Expect it to land on `master` later today, THANKS!

**Pull request closed without merging**